### DISCUSSION TITLE
ControlNET error started today

### DISCUSSION THREAD

**Discussion by meunumerotim333** [TheLastBen/fast-stable-diffusion#1661] [Q&A]
Hey guys, I'm facing this error here starting today. Already tried 3 different collabs starting from fresh install:

![image](https://user-images.githubusercontent.com/121369033/222308868-6abfc236-4db4-473e-a570-f9df88c9489b.png)

It's a general problem? Those files on the error message are inside the folder properly.

**Comment by cutecoelho** [comment]
same here

**Comment by fishonainternet** [comment]
I'm also having the same exact problem. ControlNet was working fine yesterday but now its stopped working with the same exact message as OP.

**Comment by TheLastBen** [comment]
fixed https://github.com/TheLastBen/fast-stable-diffusion/commit/0e6c82ac7e90186ecdd760553dad120a0b0dfa58

**Reply by meunumerotim333** [reply]
thank you :))

**Reply by MasterDenis** [reply]
did you also fix the issue where it wouldn't work aside from the img2img? (on yesterday's build it wouldn't even work on img2img, I did a clean install and everything) @TheLastBen

**Reply by TheLastBen** [reply]
Do you use chrome ?

**Reply by MasterDenis** [reply]
> Do you use chrome ?

tried both chrome, brave and edge, why?

**Reply by TheLastBen** [reply]
doesn't work on brave, and I think you'll get the optimal results with txt2img

**Reply by MasterDenis** [reply]
> doesn't work on brave, and I think you'll get the optimal results with txt2img

that will be an issue because it only worked on img2img as others mentioned, which browsers do you recommend because mentioning in which ones doesnt work does not help much with the situation, ideally it should work on all so there is a bug there.

**Reply by TheLastBen** [reply]
it works well with chrome, and I don't think this issue is related to colab
